src/CommonITILValidation.php

```
@@ -969,201 +969,162 @@ public function showSummary(CommonDBTM $item)
-        echo htmlspecialchars(self::getValidationStats($tID));
-        echo "</th>";
-        echo "</tr>";
-
-        echo "<tr class='tab_bg_1'>";
-        echo "<td>" . __s('Minimum validation required') . "</td>";
-        if ($canadd) {
-            echo "<td>";
-            echo $item->getValueToSelect(
-                'validation_percent',
-                'validation_percent',
-                $item->fields["validation_percent"]
-            );
-            echo "</td>";
-            echo "<td><input type='submit' name='update' class='btn btn-outline-secondary' value='" .
-                    _sx('button', 'Save') . "'>";
-            if (!empty($tID)) {
-                echo "<input type='hidden' name='id' value='$tID'>";
-            }
-            echo "</td>";
-        } else {
-            echo "<td colspan='2'>";
-            echo Dropdown::getValueWithUnit($item->fields["validation_percent"], "%");
-            echo "</td>";
-        }
-        echo "</tr>";
-        echo "</table>";
-        if ($canadd) {
-            Html::closeForm();
-        }
-
         $iterator = $DB->Request([
             'FROM'   => $this->getTable(),
             'WHERE'  => [static::$items_id => $item->getField('id')],
             'ORDER'  => 'submission_date DESC'
         ]);
 
-        $colonnes = ['', _sx('item', 'State'), __s('Request date'), __s('Approval requester'),
-            __s('Request comments'), __s('Approval status'),
-            __s('Requested approver type'), __s('Requested approver'),
+        $columns = ['', _x('item', 'State'), __('Request date'), __('Approval requester'),
+            __('Request comments'), __('Approval date'),
+            __('Requested approver type'), __('Requested approver'), __('Approval Comment'), __('Documents')
         ];
-        $nb_colonnes = count($colonnes);
+        $nb_columns = count($columns);
+        $transmessage = sprintf(__('%1$s = %2$s'), $item->getTypeName(1), $item->fields["name"]);
```

Seems unused except passing to a template that doesn't use it